feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SearchSessionsFragment.kt

```diff
@@ -148,10 +153,27 @@ class SearchSessionsFragment : Fragment(R.layout.fragment_search_sessions), Inje
         }
     }
 
+    override fun onResume() {
+        super.onResume()
+        val intent = activity!!.intent
+        if (intent.action == Intent.ACTION_SEARCH) {
+            val query: String = intent.getStringExtra(SearchManager.QUERY)!!
+            val searchResult = searchSessionsViewModel.uiModel.requireValue().searchResult
```

Comment:
I think we don't need to check to change query  😄

Reply:
I thought so.
However,  `onResume` is called after `setQuery` then it's query is same.
I want to only use once `setQuery` but  I have no idea w/o `query != searchResult.query`. 😂

Reply:
It's not so many times, so I think it's okay to call twice, but is there any problem? 👀

Reply:
Not twice. It loops  `onResume` -> `setQuery` -> `onResume`. 
For example, below code.
```
if (searchView != null) {
                searchView.setQuery(query, true)
            }
```